bundles/org.openhab.binding.linuxinput/src/main/java/org/openhab/binding/linuxinput/internal/evdev4j/jnr/EvdevLibrary.java

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright (c) 2013 Red Hat, Inc.
```

Comment:
This is a no go. All source files should have openHAB copyright. Not in the last place because we have configured the pom that it can automatically set the license header in files. This means at any point this license would be overridden here.

Reply:
That is unfortunate. But I don't think I can remove it.
Can we have two copyright headers?

Reply:
I think the same problem affects the oshi library that is currently distributed by openHAB as part of the systeminfo binding. It contains a jna clone of [libudev.h](https://github.com/oshi/oshi/blob/master/oshi-core/src/main/java/oshi/jna/platform/linux/Udev.java) which itself is LGPL-2.1+

Reply:
@Hilbrand The GPL forbids to relicense. We are in a dilemma here. I guess we need to teach SAT to ignore certain files maybe by file pattern and configure the maven plugin as well. But I can remember that Kai once said that GPL is alright for the addons2 repo.

Reply:
Before getting into a discussion about what is possible and what not. Where did this file come from? Is it just a mapping of a native interface you've created yourself or did you copy it somewhere from the internet? (I could not find it) If it's just a wrapper of the native you've created yourself than I don't see why it should contain someone else copyright.

Reply:
https://github.com/whot/libevdev/blob/master/libevdev/libevdev.h

Reply:
Not generated, written manually.
(The constants come from linux/input.h).
Ok, so I put the standard copyright blurb and leave it out of the NOTICE, too?
The only third party stuff I use is in interface definitions.

Reply:
Yup exactly

Reply:
Done